commit 692231f1bc1d7dd440b947715988cc61e4dfa10a
Author: Nicolas Berbiche <nicolas@normie.dev>
Date:   Tue Apr 5 02:12:29 2022 -0400

    xinit: use configuration time variable to retrieve libX11 so

diff --git a/ulauncher/utils/xinit.py b/ulauncher/utils/xinit.py
index d358533..cd51803 100644
--- a/ulauncher/utils/xinit.py
+++ b/ulauncher/utils/xinit.py
@@ -11,7 +11,7 @@ if IS_X11_COMPATIBLE:
     # discussion of this ABI version back to before 2009, 12 years prior to when this code was added.
     # Also, with most of the development focus on Wayland, the chance of a ABI version bump in X11 is
     # exceptionally small. Therefore, no additional fancy discovery methods are necessary.
-    x11 = cdll.LoadLibrary("libX11.so.6")
+    x11 = cdll.LoadLibrary("@libx11_path@")
 
     # In order to use any X11 calls from multiple threads, the X11 library must be initialized to use
     # mutex protection. XInitThreads() sets this up and must be called before any other X11
