commit f6ab474f8737d9071bd71d121a64209f38d6830a
Author: Nicolas Berbiche <nic.berbiche@gmail.com>
Date:   Mon May 31 17:28:28 2021 -0400

    Print invalid input buffer and continue processing

diff --git a/main.c b/main.c
index 957ec8b..f078156 100644
--- a/main.c
+++ b/main.c
@@ -812,11 +812,8 @@ main(int argc, char **argv)
 
 					old_colors = colors;
 					if (!wob_parse_input(input_buffer, &percentage, &colors.background, &colors.border, &colors.bar)) {
 						wob_log_error("Received invalid input");
-						if (!hidden) wob_hide(&app);
-						wob_destroy(&app);
-
-						return EXIT_FAILURE;
+						continue;
 					}
 
 					if (percentage > maximum) {
