diff --git a/config.el b/config.el
index 6bd59c5..adb848f 100644
--- a/config.el
+++ b/config.el
@@ -55,7 +55,7 @@
 (setq treemacs-read-string-input 'from-minibuffer)
 
 ;; Set envrc global package through config variable
-(setq envrc-direnv-executable "direnv")
+(setq envrc-direnv-executable "@envrc_direnv_package@")
 
 
 ;; Graciously donated by my friend: https://github.com/starcraft66/os-config/commit/9109152ea81c9398cc0a0d90a3728b44301ef56f
